src/main/java/Reply.java

```diff
@@ -0,0 +1,42 @@
+public class Reply {
+    public static final String PARTITION_LINE = "____________________________________________________________";
+
+    public static void printLine() {
+        System.out.println(PARTITION_LINE);
+    }
+    public static void printReply(String reply) {
+        System.out.println(PARTITION_LINE);
```

Comment:
```suggestion
        printLine();
```
And for all further occurrences as well.

Reply:
Thanks for pointing this out!